apps/codelab/src/app/components/slides/closing-slide/codelab-closing-slide.component.ts

```diff
@@ -10,7 +12,17 @@ export class CodelabClosingSlideComponent implements OnInit {
   @Input() body: String;
   @Input() footer: String;
 
-  constructor() {}
+  constructor(
+    @Optional() private readonly presentation: SlidesDeckComponent,
```

Comment:
Add a comment on why it's optional?
I'm actually curious what the use case would be, since it's always supposed to be inside of the slide deck

Reply:
ng test was failing so made it optional.